dolphinscheduler-ui/src/views/security/user-manage/components/authorize-modal.tsx

```diff
@@ -43,14 +50,14 @@ const props = {
     default: 'auth_project'
   }
 }
-
+let projectIds: string
```

Comment:
why define the projectIds here not inside the setup?

Reply:
In fact, I'm not particularly familiar with the front-end, so this part of the front-end code implementation I asked a friend to help. Regarding this declaration, I noticed that putting it in setup may causes errors at lines 116,119, and 122. If you have a better implementation, please do not hesitate to comment.

Reply:
Pass it to the render function by the return of the setup, then you can use it in the render. But I think it's better to call the functions(such as `revokeProjectByIdRequest `) in the setup function.

Reply:
I tried to rewrite this part according to your suggestion. However, maybe due to my poor front-end skill, after changing this, functions such as revokeProjectByIdRequest will get projectIds as undefined value, I tried but couldn't fix this problem.

Reply:
@Amy0104 Suggestions received and have been revised as suggested in the latest commit. I define the _projectIds_ inside the setup, and call the functions(such as _revokeProjectByIdRequest_) in the _setup_ function. Please review again.